.github/workflows/build-and-test.yml

```diff
@@ -43,8 +46,19 @@ jobs:
       - name: Build
         run: dotnet build ./lewee.sln --configuration Release --no-restore --nologo
 
-      - name: Test
-        run: dotnet test ./lewee.sln --configuration Release --no-build --nologo --settings coverage.runsettings --results-directory ./coverage
+      - name: Test (Unit Tests Only for Coverage)
+        run: |
+          # Create coverage directory
+          mkdir -p ./coverage
+          # Run unit tests for coverage to avoid third-party library coverage
+          dotnet-coverage collect "dotnet test tests/Lewee.Domain.Tests.Unit/Lewee.Domain.Tests.Unit.csproj --configuration Release --no-build --nologo" --include-files "src/Lewee.*/bin/Release/net9.0/Lewee.*.dll" -f cobertura -o ./coverage/domain-coverage.cobertura.xml
+          dotnet-coverage collect "dotnet test tests/Lewee.Application.Tests.Unit/Lewee.Application.Tests.Unit.csproj --configuration Release --no-build --nologo" --include-files "src/Lewee.*/bin/Release/net9.0/Lewee.*.dll" -f cobertura -o ./coverage/application-coverage.cobertura.xml
+          dotnet-coverage collect "dotnet test tests/Lewee.Shared.Tests.Unit/Lewee.Shared.Tests.Unit.csproj --configuration Release --no-build --nologo" --include-files "src/Lewee.*/bin/Release/net9.0/Lewee.*.dll" -f cobertura -o ./coverage/shared-coverage.cobertura.xml
+          dotnet-coverage collect "dotnet test sample-tests/Pizzeria.Store.Domain.Tests/Pizzeria.Store.Domain.Tests.csproj --configuration Release --no-build --nologo" --include-files "src/Lewee.*/bin/Release/net9.0/Lewee.*.dll" -f cobertura -o ./coverage/pizzeria-coverage.cobertura.xml
```

Comment:
There's got to be a better way to collect coverage starts than running dotnet test on each test project.

This solution doesn't scale as we don't want to change the build each time we add a new test project.